hack/tools/tilt-prepare/main.go

```diff
@@ -157,6 +157,9 @@ func main() {
 	// This makes it easier to see what clusterctl is doing and to debug it.
 	logf.SetLogger(logf.NewLogger(logf.WithThreshold(pointer.Int(5))))
 
+	// Set controller-runtime logger as well.
+	ctrl.SetLogger(klog.Background())
```

Comment:
Without this at some point CR complain that no logger has been set. We do this in all our other binaries as well